File: 백준/Bronze/5575. 타임 카드/타임 카드.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static StringTokenizer st;
    static StringBuilder sb = new StringBuilder();

    public static void main(String[] args) throws IOException {

        for (int i = 0; i < 3; i++) {
            st = new StringTokenizer(br.readLine());
            int startH = Integer.parseInt(st.nextToken());
            int startM = Integer.parseInt(st.nextToken());
            int startS = Integer.parseInt(st.nextToken());
            int endH = Integer.parseInt(st.nextToken());
            int endM = Integer.parseInt(st.nextToken());
            int endS = Integer.parseInt(st.nextToken());

            int answerH = endH - startH;
            int answerM = endM - startM;
            int answerS = endS - startS;

            if (answerS < 0) {
                answerS += 60;
                answerM -= 1;
            }
            if (answerM < 0) {
                answerM += 60;
                answerH -= 1;
            }

            System.out.println(answerH + " " + answerM + " " + answerS);
        }

    } // end of main

} // end of class

File: 백준/Bronze/5575. 타임 카드/타임 카드.py
"""
"""
import sys

input = lambda: sys.stdin.readline().strip()

for _ in range(3):
    start_h, start_m, start_s, end_h, end_m, end_s = map(int, input().split())
    answer_h = end_h - start_h
    answer_m = end_m - start_m
    answer_s = end_s - start_s
    if answer_s < 0:
        answer_s += 60
        answer_m -= 1
    if answer_m < 0:
        answer_m += 60
        answer_h -= 1
    print(answer_h, answer_m, answer_s)